README.md

```diff
@@ -42,16 +45,28 @@ kubectl create secret generic oci-volume-provisioner \
 
 ## Deploy the OCI Volume Provisioner
 
+First select the release to deploy. These are listed here. (https://github.com/oracle/oci-volume-provisioner/releases/latest)
+
+if we had selected version 1.0.0 to deploy we would execute the following.
```

Comment:
Maybe remove this line. Replace the following with the link with a placeholder in i.e. <VERSION>, and follow this with a concrete example using version 1.0.0?